tests/test_pre.py

```diff
@@ -148,5 +149,71 @@ def test_create_missing_mask(missing, order, rows, cols, expected):
     actual = pre.create_missing_mask(missing, order, rows, cols)
     np.testing.assert_array_equal(actual, expected)
 
+
+@pytest.fixture
+def test_image_files_montage(request):
+    def make_test_montage_files(missing_fields):
+        shape = (2, 2)
+
+        fields = list(range(0, 25))
+        for missing_field in missing_fields:
+            fields.remove(missing_field)
+
+        ims = [np.ones(shape, np.uint8) * i for i in fields]
+        files = []
+
+        for field, im in zip(fields, ims):
+            prefix = "MFGTMP_140206180002_A01f{0:02d}d0".format(field)
+            f, fn = tempfile.mkstemp(prefix=prefix, suffix=".tif")
+            files.append(fn)
+
+            with warnings.catch_warnings():
+                warnings.simplefilter("ignore")
+                mio.imsave(fn, im)
+
+        def cleanup():
+            for file in files:
+                os.remove(file)
+        request.addfinalizer(cleanup)
+
+        return files
+    return make_test_montage_files
+
+
+def test_montage_with_missing_montage(test_image_files_montage):
```

Comment:
I think I'll just drop the "montage" at the end of the function name, I think it's pretty clear what the test is doing :)